web/components/TextBlock/TextAndImage/TextAndImage.tsx

```diff
@@ -23,7 +24,7 @@ export const TextAndImage = ({
           {/* eslint-disable-next-line @next/next/no-img-element */}
           <img
             src={imageUrlFor(image).ignoreImageParams().url()}
-            alt=""
+            alt={image.alt}
```

Comment:
Correct me if I'm wrong, but it looks like React's "conditional rendering" means the alt attribute will be completely missing if `image.alt` is undefined, and that this will happen if the field is empty.

Given that there actually is a field for filling in a text alternative, I think we should set an empty attribute if the field is empty, so the editors have the ability to leave it empty for images that are ancillary or purely decorative. Omitting the attribute entirely is only appropriate in a couple of very specific circumstances.

```suggestion
            alt={image.alt || ''}
```

Reply:
Good catch!